Services/TimesheetsExporter.php

```diff
@@ -26,18 +26,36 @@ public function __construct(
      *
      * @return array
      */
-    public function generateData(array $criteria): array
-    {
-        $projectId = (int) ($criteria['project'] ?? 0);
-        $kind = $criteria['kind'] ?? null;
-        $dateFrom = $this->getDateTime($criteria['dateFrom'] ?? null, new Carbon('2000-01-01'));
-        $dateTo = $this->getDateTime($criteria['dateTo'] ?? null, new Carbon('2100-01-01'));
-        $userId = (int) ($criteria['userId'] ?? 0);
-        $invEmpl = filter_var($criteria['invEmpl'] ?? null, FILTER_VALIDATE_BOOLEAN) ? '1' : '';
-        $invComp = filter_var($criteria['invComp'] ?? null, FILTER_VALIDATE_BOOLEAN) ? '1' : '';
-        $paid = filter_var($criteria['paid'] ?? null, FILTER_VALIDATE_BOOLEAN) ? '1' : '';
-        $clientId = (int) ($criteria['clientId'] ?? 0);
-
-        return $this->timesheets->getAll($projectId, $kind, $dateFrom, $dateTo, $userId, $invEmpl, $invComp, $paid, $clientId, 0);
+    public function generateData(array $criteria): array {
```

Comment:
Remember the coding standards: https://github.com/ITK-Leantime/leantime-dataexport?tab=readme-ov-file#coding-standards

Reply:
Added